### PR TITLE
Add an error message for setting TTL when the storagegroup does not exist

### PR DESCRIPTION
**Pull request by RYH61**
before：
IoTDB> set ttl to root.sgcc 3600
Msg: 500: [INTERNAL_SERVER_ERROR(500)] Exception occurred: "set ttl to root.sgcc 3600". executeStatement failed. error code: TSStatus(code:500, message:322: null)

after：
IoTDB> set ttl to root.sgcc 3600
Msg: 500: [INTERNAL_SERVER_ERROR(500)] Exception occurred: "set ttl to root.sgcc 3600". executeStatement failed. error code: TSStatus(code:500, message:322: StorageGroup does not exist)

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java [hunk 1]
```diff
@@ -204,6 +204,7 @@ public TSStatus setTTL(SetTTLReq req) {
             .getStorageGroupSchema()
             .setTTL(req.getTTL());
         result.setCode(TSStatusCode.SUCCESS_STATUS.getStatusCode());
+        result.setMessage("StorageGroup does not exist");
```
Comment (wangchao316) [inline confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java:207]: TSStatusCode.SUCCESS_STATUS
and 
result.setMessage("StorageGroup does not exist");

why？

**Comment by wangchao316** [comment]
please see  [IOTDB-3259] fix sg already exists check。

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java (hunk 1 above)
Comment (RYH61) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java:207]: My mistake

Comment (RYH61) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java]: fixed~

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**